scripts/genbindings.sh

```diff
@@ -28,6 +28,24 @@ function checkInstalledLocally() {
   fi
 }
 
+function compileAndMoveTo() {
+  if [ ! -d "$1" ] || [ ! -d "$2" ]; then
+    echo ERROR: No valid source or destination specified
+    echo compileAndMoveTo [source] [destination]
+    exit 1
+  fi
+
+  CWD="${PWD}"
+  cd "${1}"
+
+  mkdir -p "${2}"/contracts
+  mkdir -p "${2}"/contracts/abi
+  npx hardhat compile && cp -r ./typechain/* "${2}"/contracts/ \
+    && find ./artifacts/contracts/*.sol/ \
+    | grep -v ".dbg" | grep ".json" | xargs cp -t "${2}"/contracts/abi/
```

Comment:
again, can we move?

Reply:
Now `cp -rl`'ing the required generated bindings/abis. Forced us to remove existing bindings beforehand, which had the nice side effect of removing out of date openzeppelin contracts.